pkg/crossplane/service_mariadb_database.go

```diff
@@ -243,7 +246,7 @@ func mapMariadbEndpoint(data map[string][]byte) (*Endpoint, error) {
 	}, nil
 }
 
-func createCredentials(endpoint *Endpoint, username, password, database, metricsPort string) Credentials {
+func createCredentials(endpoint *Endpoint, username, password, database, cn string, metricsEnabled bool, md string) Credentials {
```

Comment:
Minor nitpick: `cn` and `md` cloud be a bit more self explanatory. I.e. `clusterName` and `metricsDomain`